package.json

```diff
@@ -66,10 +66,14 @@
     "tape-run": "3.0.0",
     "tslint": "^5.2.0",
     "tslint-config-standard": "^6.0.0",
-    "typescript": "^2.1.4",
-    "typings": "^2.0.0"
+    "typescript": "^2.1.4"
   },
   "dependencies": {
+    "@types/concat-stream": "^1.6.0",
+    "@types/form-data": "0.0.33",
+    "@types/methods": "^1.1.0",
+    "@types/node": "^8.0.0",
```

Comment:
Can we move `@types/node` to dev dependencies? Installing `node` for people using the browser could cause issues.